### PR TITLE
Error correction in first 2 lines

### PR DESCRIPTION
**Pull request by dangerouspython**
The first two lines had a few errors which were corrected

http://www.poetryfoundation.org/poem/173536

### CODE REVIEW

File: road_not_taken.txt [hunk 1]
```diff
@@ -1,5 +1,5 @@
-Two roads diverged in a wood,
-And sorry I coud travel both
+Two roads diverged in a yellow wood,
+And sorry I could not travel both
 And be a traveller, long I stood
```
Comment (malignantpython) [inline road_not_taken.txt]: If you could change 'a' to 'one' in the 3rd line, it would be good

**Comment by malignantpython** [comment]
Then I can merge your commit

**Comment by dangerouspython** [comment]
Ok.. I will do it

**Comment by dangerouspython** [comment]
I also noticed an error in the 4th line which I corrected. Anything else you would like me to do?

**Comment by malignantpython** [comment]
Merged the pull request

**Comment by dangerouspython** [comment]
- [x] Hello
- [ ] Die!
- [ ] Get lost

**Comment by dangerouspython** [comment]
``` c++
cin >> user
```

**Comment by dangerouspython** [comment]
``` c++
for (int i = 0; i < 4; i++)
{
      cout << i;
}
```

**Comment by dangerouspython** [comment]
``` python
def class New:
    pass
```

**Comment by dangerouspython** [comment]
> Two roads diverged in a yellow wood,
> And sorry I could not travel both

**Comment by dangerouspython** [comment]
:smile: :fireworks:

**Pull request merged**